frontend/public/components/machine.tsx

```diff
@@ -149,52 +149,68 @@ const MachineDetails: React.SFC<MachineDetailsProps> = ({ obj }: { obj: MachineK
               <ResourceSummary resource={obj} />
             </div>
             <div className="col-sm-6">
-              <DetailsItem label="Phase" obj={obj} path="status.phase">
-                <Status status={getMachinePhase(obj)} />
-              </DetailsItem>
-              <DetailsItem
-                label="Provider State"
-                obj={obj}
-                path="status.providerStatus.instanceState"
-              >
-                {providerState}
-              </DetailsItem>
+              <dl className="co-m-pane__details">
+                <DetailsItem label="Phase" obj={obj} path="status.phase">
+                  <Status status={getMachinePhase(obj)} />
+                </DetailsItem>
+              </dl>
+              <dl className="co-m-pane__details">
```

Comment:
You shouldn't need a separate `dl` around each item, just one around the entire list

Reply:
You're completely right! I thought I was running into issues with that earlier, but I think I must have just put the overall `dl` in the wrong location before. Sorry about that! :)